pkg/ctl/scale/nodegroup.go

```diff
@@ -19,18 +25,27 @@ func scaleNodeGroupCmd(cmd *cmdutils.Cmd) {
 
 	cmd.CobraCommand.RunE = func(_ *cobra.Command, args []string) error {
 		cmd.NameArg = cmdutils.GetNameArg(args)
-		return doScaleNodeGroup(cmd, ng)
+		return runFunc(cmd, ng)
 	}
 
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
 		fs.StringVar(&cfg.Metadata.Name, "cluster", "", "EKS cluster name")
 		fs.StringVarP(&ng.Name, "name", "n", "", "Name of the nodegroup to scale")
 
-		desiredCapacity := fs.IntP("nodes", "N", -1, "total number of nodes (scale to this number)")
+		desiredCapacity := fs.IntP("nodes", "N", -1, "desired number of nodes (required)")
+		maxCapacity := fs.IntP("nodes-max", "M", -1, "maximum number of nodes")
+		minCapacity := fs.IntP("nodes-min", "m", -1, "minimum number of nodes")
```

Comment:
I'm not sure the shorthand flags `M` and `m` are intuitive. If you think they're useful, we should ensure to use only the longer-name flags in examples.

Reply:
Oh I just want to keep consistency with https://github.com/weaveworks/eksctl/blob/master/pkg/ctl/cmdutils/nodegroup_flags.go#L17. Agreed that the short hands here are a little bit ambiguous, let me know what you want to update here. Thanks

Reply:
I think we can keep the shorthand version for consistency.